pkg/ingester/circuitbreaker.go

```diff
@@ -48,6 +49,11 @@ func newCircuitBreakerMetrics(r prometheus.Registerer, currentState func() circu
 			Help:        "Results of executing requests via the circuit breaker.",
 			ConstLabels: map[string]string{circuitBreakerRequestTypeLabel: requestType},
 		}, []string{"result"}),
+		circuitBreakerRequestTimeouts: promauto.With(r).NewCounter(prometheus.CounterOpts{
+			Name:        "cortex_ingester_circuit_breaker_request_timeouts_total",
+			Help:        "Number of times a circuit breaker request reached a timeout.",
```

Comment:
Is it a "circuit breaker request"? Or rather a request that reached timeout, that we registered into circuit breaker.

Reply:
I have changed it into "Number of times the circuit breaker recorded a request that reached timeout."